### PR TITLE
Generate templates in tests folder

### PR DESCRIPTION
**Pull request by mboukhalfa**
and rename test file since it does not need the prefix order number

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ host_cleanup:
 	./host_cleanup.sh
 
 test:
-	./tests/05_test.sh
+	./tests/test.sh
```
Comment (furkatgofurov7) [inline Makefile:24]: but `test.sh` does not exist under the `tests` folder: https://github.com/metal3-io/metal3-dev-env/tree/main/tests

File: tests/roles/run_tests/vars/main.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ NAMESPACE: "{{ lookup('env', 'NAMESPACE') | default('metal3', true) }}"
 IRONIC_NAMESPACE: "{{ lookup('env', 'IRONIC_NAMESPACE') | default('baremetal-operator-system', true) }}"
 NAMEPREFIX: "{{ lookup('env', 'NAMEPREFIX') | default('baremetal-operator', true) }}"
 CRS_PATH: "{{ metal3_dir }}/tests/roles/run_tests/templates"
-TEMP_GEN_DIR: "{{ metal3_dir }}/vm-setup/roles/run_tests/files/manifests"
+TEMP_GEN_DIR: "{{ metal3_dir }}/tests/roles/run_tests/files/manifests"
 MANIFESTS_DIR: "{{ metal3_dir }}/vm-setup/roles/run_tests/files"
```
Comment (furkatgofurov7) [inline tests/roles/run_tests/vars/main.yml:9]: what are these folders for and why one `MANIFESTS_DIR` is different from the other `TEMP_GEN_DIR` with this change?

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: tests/roles/run_tests/vars/main.yml (hunk 2 above)
Comment (mboukhalfa) [reply tests/roles/run_tests/vars/main.yml:9]: MANIFESTS_DIR was updated already and it is the path to all manifests I think and TEMP_GEN_DIR is one contains the generated templates

File: Makefile (hunk 1 above)
Comment (mboukhalfa) [reply Makefile:24]: yes I changing `05_test.sh` that exists to `test.sh` I mean just renaming a file

Comment (furkatgofurov7) [reply Makefile:24]: I am not sure if I understand this

File: tests/roles/run_tests/vars/main.yml (hunk 2 above)
Comment (furkatgofurov7) [reply tests/roles/run_tests/vars/main.yml:9]: but the paths are same, we do not have `run_tests` folder under `vm-setup/roles` anymore AFAIU, why the `MANIFESTS_DIR` should not be changed?

File: Makefile (hunk 1 above)
Comment (furkatgofurov7) [reply Makefile:24]: ah I overlooked file renaming change completely, sorry

File: tests/roles/run_tests/vars/main.yml (hunk 2 above)
Comment (mboukhalfa) [reply tests/roles/run_tests/vars/main.yml:9]: nice catch, I will check this. I was thinking that  already under `tests`

Comment (mboukhalfa) [reply tests/roles/run_tests/vars/main.yml]: That var `MANIFESTS_DIR` is not used anywhere so I think we do not needed ! removed is that ok ?

**Review by furkatgofurov7 — APPROVED** [review]
/cc @lentzi90 @Sunnatillo @smoshiur1237

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Pull request merged**